### ISSUE TITLE
[Bug]: Turning "showToughnessValue" to false doesn't actually remove the text value next to the Toughness Bar.

### ISSUE THREAD

**Issue by Gbergz** [Cheos137/ArmorpointsPlusplus#24]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Currently have this: (useToughnessBar is enabled!)

```
	# Show toughness value text next to toughness bar
	# Available: true, false [default: true]
	showToughnessValue = false
```

but the text next to toughness bar is still being rendered despite being disabled.
disabling this option for health works, but not for toughness.

### Version

Other (this list is not up to date - again...)

### Version

forge-1.18.2-3.0.0

### To Reproduce

1. Set `useToughnessBar` from `false` to `true`
1. Turn `showToughnessValue` from `true` to `false`
2. Start game, go into a world.
3. Put on some armor with toughness values.
4. Text still being rendered.
Profit.

### Expexted Behavior

That the text is disabled next to the Toughness bar.

### Additional Context

Edit: Oh while I'm here, would it be possible to add the colored health to ingame tab?
Currently use this on servers to display everybody's total health on TAB.
(**/scoreboard objectives add h health -- then use -- /scoreboard objectives setdisplay list h**)

### Screenshots

![image](https://user-images.githubusercontent.com/26634554/166116794-b277c9f8-f21c-417f-9b2b-6e69546ca594.png)

### Relevant Log / Crash-Report Output

```shell
Nothing to show in logs.
```

**Comment by Cheos137** [comment]
fixed in 3.0.1 (98bf1a5)

also i've opened a feature request for the scoreboard thingy

**Comment by Gbergz** [comment]
Awesome thanks for the fix.
And thanks for bringing the feature request forwards. :)
